### ISSUE TITLE
ServerSocket.close()

### ISSUE THREAD

**Issue by Rikud** [alesiatants/Client_Server_Host_To_IP_With_Threading#6]
https://github.com/alesiatants/Client_Server_Host_To_IP_With_Threading/blob/b453210548c875794fe4780595cf8e95a5383cae/Client_Server_DNS_IP_Thread/server.py#L54

выполнение никогда не доберется до этой строчки кода

**Comment by alesiatants** [comment]
Да, я знаю. Однако данную строку можно было бы определить в дальнейшем в качестве блока, выполняющегося при положительной проверке условия по завершению работы сервера.
